qutebrowser/mainwindow/tabwidget.py

```diff
@@ -161,6 +187,11 @@ def update_tab_titles(self):
         for idx in range(self.count()):
             self.update_tab_title(idx)
 
+    @config.change_filter('tabs', 'title-format-pinned')
+    def update_tab_titles_pinned(self):
+        """Update all texts."""
+        self.update_tab_titles()
+
```

Comment:
Oh, I never realized that the connect method passes those in if those are available, this makes a lot more sense now :P :+1: